src/machine/usb/joystick/joystick.go

```diff
@@ -1,97 +0,0 @@
-package joystick
-
-import (
-	"machine"
-	"machine/usb"
-)
-
-const (
-	jsEndpointOut = usb.HID_ENDPOINT_OUT // from PC
-	jsEndpointIn  = usb.HID_ENDPOINT_IN  // to PC
-)
-
-var js *Joystick
-
-type Joystick struct {
-	State
-	buf           *RingBuffer
-	waitTxc       bool
-	rxHandlerFunc func(b []byte)
-	setupFunc     func(setup usb.Setup) bool
-}
-
-func Enable(def Definitions, rxHandlerFunc func(b []byte),
-	setupFunc func(setup usb.Setup) bool, hidDesc []byte) *Joystick {
-	m := &Joystick{
-		buf:   NewRingBuffer(),
-		State: def.NewState(),
-	}
-	m.State = def.NewState()
-	if setupFunc == nil {
-		setupFunc = m.setupFunc
-	}
-	machine.EnableJoystick(m.handler, rxHandlerFunc, setupFunc, hidDesc)
-	js = m
-	return m
-}
```

Comment:
This is a necessary part of ForceFeedbackJoystick.
I'll consider a few ideas on how to change it.

Reply:
I would like you to MERGE the following

https://github.com/tinygo-org/tinygo/commit/f3efea9e7a65f50ae29c31e22a94ce6a0bb87351


This allows the ForceFeedbackJoystick to be configured.
The following is the code that will be changed by this PR.

https://github.com/sago35/pico-ffb-steering-wheel/commit/e737234a42ace757eae7efaa85f245ffbf86b54d

Reply:
I had a feeling you were using this somewhere. See my suggestion on small rename to your proposed function.

Reply:
Updated

https://github.com/tinygo-org/tinygo/tree/usb-hid-joystick-refactor-sago35